client/me/mcp/main.jsx

```diff
@@ -231,6 +240,36 @@ function McpComponent( { path, userSettings, isUpdating } ) {
 					</>
 				) }
 
+				{ hasTools && anyToolsEnabled && (
+					<>
+						<div style={ { marginTop: '24px' } }>
```

Comment:
Let's use `VStack` to control the spacing. Also, the outer fragment doesn't seem to be required.